content/en/service_management/workflows/datastore.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Apps Datastore
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1gVVWLg5P3EXkVXBc4qZtqKp_nqykDsFXCMmh6pbvxx0/edit" btn_hidden="false" header="Try the Preview!">}}
+Datastore is in Preview. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+The Apps Datastore offers a scalable, structured data storage solution within Datadog's App Builder and Workflow Automation products. It supports CRUD (Create, Read, Update, and Delete) operations and integrates seamlessly with Datadog's ecosystem to optimize persistent data storage without the need for external databases.
+
+## Create a datastore
+
```

Comment:
A few general comments:
1. Do you think it is also important to mention that it is possible to search the table using the search field?
2.Do you think it is also important to mention that it is possible sort the table by every column by clicking on the column name?
3. I think it is also important to explain how to interact with the datastore from an App / WF and list the relevant actions

Reply:
1. I'll take a look.
2. This is already explained in the "reference" section.